set.mm

```diff
@@ -94,6 +94,13 @@ proposed  syl       imtri       (analogous to *bitr*, sstri, etc.)
 
 DONE:
 Date      Old       New         Notes
+20-Dec-21 bj-eleq1w eleq1w      moved from BJ's mathbox to main set.mm
+20-Dec-21 rabeqi    [same]      moved from GS's mathbox to main set.mm
+20-Dec-21 rabeqif   [same]      moved from GS's mathbox to main set.mm
```

Comment:
why not "rabeqfi" ?

Reply:
I also worried about that, so I took what Glauco used. I think there is no rule in which order to append the suffixes "i" and "f". Using "if" could cause confusion with the conditional operator or "dif", but "fi" is also used as suffix for "finite". Since we have both rabeqf and rabeqi, we can apply the rule in both ways. I found no other example for such a combination...

Reply:
As @tirix wrote in his comment, the "f" is more often placed after the "i", "d" or "g" and therefore shoud become a rule/convention. I will add it to our conventions in set.mm in my next PR.

In this special case, rabeqif will not be changed.